AWS Lambda: panic: cannot statfs cgroup root: no such file or directory

**Describe the bug**
Running in AWS lambda, the application cannot start due to `panic: cannot statfs cgroup root: no such file or directory` from `go/pkg/mod/github.com/zeromicro/go-zero@v1.3.4/core/stat/internal/cgroup_linux.go:191`

See [opencontainers/runc#2634](https://github.com/opencontainers/runc/pull/2634/files) - they don't execute this code on `init`

**To Reproduce**
Steps to reproduce the behavior, if applicable:

1. The code is

```go
package main

import (
	"context"
	"github.com/aws/aws-lambda-go/lambda"
	"github.com/zeromicro/go-zero/core/stat"
)

func main() {
	stat.CpuUsage()
	lambda.Start(eventHandler)
}

type Event struct{}

func eventHandler(ctx context.Context, ev Event) error {
	return nil
}
```

2. The error is

```
panic: cannot statfs cgroup root: no such file or directory
--
goroutine 1 [running]:
github.com/zeromicro/go-zero/core/stat/internal.isCgroup2UnifiedMode.func1()
/Users/josh/go/pkg/mod/github.com/zeromicro/go-zero@v1.3.4/core/stat/internal/cgroup_linux.go:191 +0x13a
sync.(*Once).doSlow(0xc00008dcf0?, 0x0?)
/opt/homebrew/Cellar/go/1.18.1/libexec/src/sync/once.go:68 +0xc2
sync.(*Once).Do(...)
/opt/homebrew/Cellar/go/1.18.1/libexec/src/sync/once.go:59
github.com/zeromicro/go-zero/core/stat/internal.isCgroup2UnifiedMode()
/Users/josh/go/pkg/mod/github.com/zeromicro/go-zero@v1.3.4/core/stat/internal/cgroup_linux.go:182 +0x31
github.com/zeromicro/go-zero/core/stat/internal.currentCgroup()
/Users/josh/go/pkg/mod/github.com/zeromicro/go-zero@v1.3.4/core/stat/internal/cgroup_linux.go:39 +0x17
github.com/zeromicro/go-zero/core/stat/internal.cpuSets()
/Users/josh/go/pkg/mod/github.com/zeromicro/go-zero@v1.3.4/core/stat/internal/cpu_linux.go:112 +0x19
github.com/zeromicro/go-zero/core/stat/internal.init.0()
/Users/josh/go/pkg/mod/github.com/zeromicro/go-zero@v1.3.4/core/stat/internal/cpu_linux.go:27 +0x1d
2022/06/14 21:51:22 exit status 2
```

**Expected behavior**
Running application

**Screenshots**


**Environments (please complete the following information):**
 - OS: Linux
 - go-zero version: 1.3.4
 - goctl version: 1.3.4

**More description**
Using Go 1.18 on AWS Lambda


The code in question is
```go
func isCgroup2UnifiedMode() bool {
	isUnifiedOnce.Do(func() {
		var st unix.Statfs_t
		err := unix.Statfs(cgroupDir, &st)
		if err != nil {
			if os.IsNotExist(err) && runningInUserNS() {
				// ignore the "not found" error if running in userns
				isUnified = false
				return
			}
			panic(fmt.Sprintf("cannot statfs cgroup root: %s", err))
		}
		isUnified = st.Type == unix.CGROUP2_SUPER_MAGIC
	})

	return isUnified
}
```

--- a/core/stat/internal/cpu_linux.go
+++ b/core/stat/internal/cpu_linux.go
@@ -4,6 +4,7 @@ import (
 	"errors"
 	"fmt"
 	"strings"
+	"sync"
 	"time"
 
 	"github.com/zeromicro/go-zero/core/iox"
@@ -22,8 +23,10 @@ var (
 	cores     uint64
 )
 
+var initonce sync.Once
+
 // if /proc not present, ignore the cpu calculation, like wsl linux
-func init() {
+func initialize() {
 	cpus, err := cpuSets()
 	if err != nil {
 		logx.Error(err)
@@ -69,6 +72,7 @@ func init() {
 
 // RefreshCpu refreshes cpu usage and returns.
 func RefreshCpu() uint64 {
+	initonce.Do(initialize)
 	total, err := totalCpuUsage()
 	if err != nil {
 		return 0

